CONTRIBUTING.md

```diff
@@ -167,6 +167,40 @@ If you are not familiar with creating a Pull Request, here are some guides:
 - http://stackoverflow.com/questions/14680711/how-to-do-a-github-pull-request
 - https://help.github.com/articles/creating-a-pull-request/
 
+#### Sync up with the upstream
```

Comment:
I'd say this is a sort of subsection of "Send a PR" right ?
```suggestion
##### Sync up with the upstream
```

Reply:
At first, I thought this for updating the forked repo with upstream.
but yes, it makes more sense under "Send a PR"